src/Poseidon/CLI/OptparseApplicativeParsers.hs

```diff
@@ -600,7 +600,8 @@ parseListEntity = parseListPackages <|> parseListGroups <|> (parseListIndividual
     parseListIndividualsDummy = OP.flag' () (
         OP.long "individuals" <>
         OP.help "List all individuals/samples.")
-    parseListIndividualsExtraCols = ListIndividuals <$> OP.many parseExtraCol
+    parseListIndividualsExtraCols = ListIndividuals <$> (parseAllJannoCols <|> (Just <$> OP.many parseExtraCol))
+    parseAllJannoCols = OP.flag' Nothing (OP.long "fullJanno" <> OP.help "output all Janno Columns")
```

Comment:
It's a bit inconsistent that for the Web-API we have a keyword to express "give me all" (`/individuals?additionalJannoColumns=ALL`), but on the command line we have a special option (`--fullJanno`).

We could also use a keyword on the command line (`--jannoColumn ALL`). Or on the API side a special option (`/individuals?fullJanno`).

But CLI and API are already inconsistent in that `--jannoColumn` takes only one column name, but can be listed multiple times, whereas `additionalJannoColumns` of the webserver takes a list of column names.

Up to you - just wanted to point it out.

Reply:
I will reply in the main conversation thread.